knowhere/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@
 include_directories(${INDEX_SOURCE_DIR}/knowhere)
 include_directories(${INDEX_SOURCE_DIR}/thirdparty)
 add_subdirectory(${INDEX_SOURCE_DIR}/knowhere/knowhere/utils)
+add_subdirectory(${INDEX_SOURCE_DIR}/knowhere/knowhere/index/vector_index/impl/bruteforce)
```

Comment:
bruteforce already is sub dir of knowwhere,  so using add_subdirectory is a best practice?

Reply:
cc @cydrain  any ideas?

Reply:
need not add_subdirectory here, you can reference how nsg does

Reply:
done.